lib/connectors/memory.js

```diff
@@ -461,6 +460,15 @@ function applyFilter(filter) {
     if (typeof value === 'string' && (example instanceof RegExp)) {
       return value.match(example);
     }
+
+    if (example.regexp) {
+      var normalizedRegExp = utils.toRegExp(example.regexp);
+      if (normalizedRegExp instanceof Error)
```

Comment:
I'm only using toRegExp here to normalize the regex because somewhere above the chain my regex is converted to a string.